<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fb2mobi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="3" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Ignored</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>124</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="checkConvertToSrcDir">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Конвертировать в исходную папку</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Конвертировать в папку</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Профиль конвертера</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="comboProfiles"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Формат</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboOutputFormat">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="editDestPath">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelectDestDir">
            <property name="text">
             <string>Выбрать</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnOpenDestDir">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextOnly</enum>
            </property>
            <property name="autoRaise">
             <bool>false</bool>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTreeWidget" name="treeFileList">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::NoDragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::IgnoreAction</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Файл</string>
        </property>
       </column>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="value">
          <number>24</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>453</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnStart">
         <property name="text">
          <string>Пуск</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionAddFile"/>
    <addaction name="actionConvert"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionOpenForumURL"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionViewLog"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAddFile">
   <property name="text">
    <string>Добавить...</string>
   </property>
   <property name="toolTip">
    <string>Добавить файлы для конвертирования</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionConvert">
   <property name="text">
    <string>Конвертировать</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionOpenForumURL">
   <property name="text">
    <string>Форум поддержки</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="actionViewLog">
   <property name="text">
    <string>Посмотреть журнал</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>closeApp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddFile</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addFilesAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConvert</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>startConvert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStart</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startConvert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSelectDestDir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectDestDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>684</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOpenDestDir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openDestDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editDestPath</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>checkDestDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkConvertToSrcDir</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>checkConvertToDestDirClicked(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteRecAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenForumURL</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openSupportURL()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionViewLog</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>closeApp()</slot>
  <slot>addFilesAction()</slot>
  <slot>startConvert()</slot>
  <slot>selectDestDir()</slot>
  <slot>openDestDir()</slot>
  <slot>checkDestDir()</slot>
  <slot>checkConvertToDestDirClicked(int)</slot>
  <slot>deleteRecAction()</slot>
  <slot>openSupportURL()</slot>
  <slot>openLog()</slot>
  <slot>about()</slot>
 </slots>
</ui>
